core/domain/topic_domain.py

```diff
@@ -1276,6 +1284,15 @@ def update_thumbnail_bg_color(self, new_thumbnail_bg_color):
         """
         self.thumbnail_bg_color = new_thumbnail_bg_color
 
+    def update_thumbnail_size_in_bytes(self, new_thumbnail_size_in_bytes):
```

Comment:
You don't need a separate function here.

Reply:
Updated, both thumbnail filename and size_in_bytes will be updated within the update_thumbnail_filename()